dask_cloudprovider/aws/ecs.py

```diff
@@ -754,6 +759,7 @@ def __init__(
         mount_volumes_on_scheduler=False,
         **kwargs,
     ):
+        self._cpu_architecture = cpu_architecture
```

Comment:
Could we add a case check here?

It looks like valid values are `X86_64` and `ARM64`. However the lowercase `x86_64` and `arm64` are also very commonly used in other situations (this is what `uname -m` will show).

Perhaps we want to either explicitly set this with `.upper()` or check the value against known good values (although if more are added in the future this adds a maintenance burden).

Reply:
.upper() it is!

Reply:
@jacobtomlinson I also moved it up in line with order of the constructor fields. Happy to proceed?